### ISSUE TITLE
Can't get the same results as paper

### ISSUE THREAD

**Issue by lioutasb** [zcyang/imageqa-san#2]
I'm trying to reproduce the results from DAQAUR-REDUCED dataset as mentioned in paper but it seems the model can't get an accuracy above 40% . The paper states an accuracy of 46.2% with SAN(2, LSTM) model. It would be nice if you could upload the pre-processed files from this dataset also and the options you used for training.

Thank you,
Vasilis Lioutas

**Comment by zcyang** [comment]
the default configuration is for vqa data set, not the daquar data set. You can directly download the vqa data (the link has been provided) and run the training, this will lead to top-1 (most frequent label or answer) accuracy of >50%, you can then evaluate the metric used for vqa data set, which should be equivalent to the result reported in the paper.

For daquar, you should reduce the model size and increase the dropout ratio to reduce overfitting.
